### DISCUSSION TITLE
How to make 2 items in one container?

### DISCUSSION THREAD

**Discussion by nrf24l01** [flet-dev/flet#1521] [General]
I try smth like `ft.Container(
                ft.Image(
                    src="static/zametki_site.png",
                    width=100,
                    height=100,
                    fit=ft.ImageFit.NONE,
                    repeat=ft.ImageRepeat.NO_REPEAT,
                    border_radius=ft.border_radius.all(10),
                ),
                ft.Text(f"{i['name']}", color=lvl_colors[i["lvl"]]['txt'],top=True),
                width=width,
                height=int(width*1.61803398875),
                bgcolor=lvl_colors[i["lvl"]]['bg'],
                border=ft.border.all(2, lvl_colors[i["lvl"]]['border']),
                border_radius=ft.border_radius.all(5),
            )` but i saw only image. How i can show 2 elements in one container?

**Comment by TingTingin** [comment]
you have to put the elements in another control like a row or column like

ft.Column([ft.Image( src="static/zametki_site.png", width=100, height=100, fit=ft.ImageFit.NONE, repeat=ft.ImageRepeat.NO_REPEAT, border_radius=ft.border_radius.all(10), ), ft.Text(f"{i['name']}", color=lvl_colors[i["lvl"]]['txt'],top=True), width=width, height=int(width*1.61803398875), bgcolor=lvl_colors[i["lvl"]]['bg'], border=ft.border.all(2, lvl_colors[i["lvl"]]['border']])

and the put the column in the container

**Comment by nrf24l01** [comment]
And if i want to make smth like 
![image](https://github.com/flet-dev/flet/assets/96012285/b326781c-a80b-44ba-b79e-aee0bbd28596)
I need to make logic like  `container(row(image), row(column(text,text,progressbar)))`?
